src/ibek/support_cmds/commands.py

```diff
@@ -56,51 +56,80 @@ class AptWhen(str, Enum):
 support_cli = typer.Typer(cls=NaturalOrderGroup)
 
 
-@support_cli.command(
-    context_settings={"allow_extra_args": True, "ignore_unknown_options": True}
-)
-def apt_install(
-    ctx: typer.Context,
-    debs: List[str] = typer.Argument(None, help="list of debian packages to install"),
-    only: AptWhen = typer.Option(
-        AptWhen.both, help="which container build stage to install in"
-    ),
-    runtime: bool = typer.Option(False, help="install list of runtime packages"),
-):
+def _install_debs(debs: List[str], args) -> None:
     """
-    Install debian packages into the container. If they have an http:// or https://
+    Install a list of debian packages.
+
+    If they have an http:// or https://
     prefix then they will be downloaded and installed from file.
     """
-    if runtime and not GLOBALS.NATIVE:
-        print("skipping runtime install in cross-compile environment")
-        return
-
     temp = Path("/tmp")
-
-    if (only is AptWhen.run) or (only is AptWhen.both):
-        add_list_to_file(RUNTIME_DEBS, debs)
-    if only is AptWhen.run:
-        return
-
-    if runtime and RUNTIME_DEBS.exists():
-        debs += RUNTIME_DEBS.read_text().split()
-
     for i, pkg in enumerate(debs):
         if pkg.startswith("http://") or pkg.startswith("https://"):
             pkg_file = temp / pkg.split("/")[-1]
             subprocess.call(["wget", pkg, "-O", str(pkg_file)])
             debs[i] = str(pkg_file)
 
+    if len(debs) == 0:
+        print("no packages to install")
+        return
+
+    print("installing packages: ", debs)
+
     sudo = "sudo" if os.geteuid() != 0 else ""
     command = (
         f"{sudo} apt-get update && {sudo} apt-get upgrade -y && "
         f"{sudo} apt-get install -y --no-install-recommends "
         + " ".join(debs)
-        + " ".join(ctx.args)
+        + " ".join(args)
     )
     exit(subprocess.call(["bash", "-c", command]))
 
 
+@support_cli.command(
+    context_settings={"allow_extra_args": True, "ignore_unknown_options": True}
+)
+def runtime_apt_install(
+    ctx: typer.Context,
+    debs: List[str] = typer.Argument(None, help="list of debian packages to install"),
```

Comment:
Do we need to install any extra debs here? I can't think of a usecase where we currently need it, so I'd argue it makes the interface less obvious by having it in there.

E.g.
`ibek runtime-apt-install somepkg` makes it sound like it would only install `somepkg` where in fact it would install all the runtime libs specified by ibek-support modules as well as `somepkg`